backend/graphql/types/testSessionType.ts

```diff
@@ -2,12 +2,16 @@ import gql from "graphql-tag";
 
 const testSessionType = gql`
   type ResultResponseDTO {
-    student: StudentResponseDTO!
     score: Float!
     answers: [[[Float]]]!
     breakdown: [[Boolean]]!
   }
 
+  type StudentResultEntryDTO {
+    student: StudentResponseDTO!
+    result: ResultResponseDTO
+  }
```

Comment:
what does `Entry` mean here? can we do `StudentResultResponseDTO` to be consistent w/ naming in other parts of codebase?